### PR TITLE
#7743: Integrate yolov4 modules

### PR DESCRIPTION
**Pull request by saichandax**
YoloV4 integration work.

### CODE REVIEW

File: models/experimental/functional_yolov4/reference/yolov4.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from models.experimental.functional_yolov4.reference.downsample1 import DownSample1
```
Comment (saichandax) [inline models/experimental/functional_yolov4/reference/yolov4.py:1]: spdx license needed

File: models/experimental/functional_yolov4/tt/ttnn_yolov4.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from models.experimental.functional_yolov4.tt.ttnn_downsample1 import TtDownSample1
```
Comment (saichandax) [inline models/experimental/functional_yolov4/tt/ttnn_yolov4.py:1]: spdx license needed

**Comment by dvartaniansTT** [comment]
@saichandax please add a README file to include how to run the demo/inference pipeline. also instructions on where to download the model weights from and where to save them.

**Comment by saichandax** [comment]
@ankitmcw ,
Please update with Readme and also fix the formatting, license issues.

**Comment by saichandax** [comment]
The PR is outdated. We have new PR for Yolov4 integration at: #10068

**Pull request closed without merging**